fe/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -118,10 +129,57 @@ public void processTimeoutJobs() {
         idToLoadJob.values().stream().forEach(entity -> entity.processTimeout());
     }
 
+    public List<List<Comparable>> getLoadJobInfosByDb(long dbId, String labelValue,
```

Comment:
add some comment for this function? Let people known what does this function do without reading code